1209/4646/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: SmartPID SPC1000
```

Comment:
Since the org name is prefixed to the title, this will show as 'SmartPID SmartPID SPC1000'; is that what you intended?